backend/src/api/rating/content-types/rating/schema.json

```
@@ -0,0 +1,36 @@
+{
+  "kind": "collectionType",
+  "collectionName": "ratings",
+  "info": {
+    "singularName": "rating",
+    "pluralName": "ratings",
+    "displayName": "Rating"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "value": {
+      "type": "integer",
+      "default": 1,
```

```suggestion
      "default": 0,
```

If it defaults to something invalid it could prevent accidental ratings.